contrib/platform/src/com/sun/jna/platform/win32/Psapi.java

```diff
@@ -320,4 +338,79 @@ class PERFORMANCE_INFORMATION extends Structure {
         public DWORD ProcessCount;
         public DWORD ThreadCount;
     }
+
+    @FieldOrder({"Flags"})
+    class PSAPI_WORKING_SET_EX_BLOCK extends Structure {
+        public ULONG_PTR[2] Flags = new ULONG_PTR[Native.POINTER_SIZE == 8 ? 1 : 2];
```

Comment:
Two more notes:
1. The type should be `ULONG_PTR[]` without the 2.  I think that's causing the checkstyle errors.
2. Thank you for this PR!